### PR TITLE
start profile picture feature

### PR DESCRIPTION
**Pull request by ramib20**

### CODE REVIEW

File: Test2/app/src/test/java/com/example/test2/ExampleUnitTest.java [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 import static org.junit.Assert.*;
 
+import org.junit.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (taehee5525) [inline Test2/app/src/test/java/com/example/test2/ExampleUnitTest.java:9]: Let's avoid using the wild card

File: Test2/app/src/main/java/com/example/test2/data/ProfilePicture.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import java.lang.Object;
+import android.content.Intent;
+import android.view.View;
+import java.net.URI;
+import android.graphics.Bitmap;
+import java.io.InputStream;
+//import
```
Comment (taehee5525) [inline Test2/app/src/main/java/com/example/test2/data/ProfilePicture.java:7]: Please remove this line if not needed

**Review by taehee5525 — APPROVED** [review]
Just a few things to change. Other than that, good work

File: Test2/app/src/main/java/com/example/test2/data/ProfilePicture.java (hunk 2 above)
Comment (kimhajiny) [reply Test2/app/src/main/java/com/example/test2/data/ProfilePicture.java:7]: agree

Comment (kimhajiny) [reply Test2/app/src/main/java/com/example/test2/data/ProfilePicture.java:7]: other than that good to be merged.

**Pull request merged**